src/components/itemlabal/ItemLabel.css

```diff
@@ -0,0 +1,26 @@
+.article{
+    display:flex;
+    flex-direction: column;
```

Comment:
Doe je even CTRL + ALT + L in dit document om alle enters en spaties goed te zetten?